kernel/enter.c

```diff
@@ -931,6 +931,8 @@ int myst_enter_kernel(myst_kernel_args_t* args)
         free(process->cwd);
         process->cwd = NULL;
 
+        free(process);
```

Comment:
Maybe set process to NULL?

Reply:
The whole kernel is shutting down at this point so I didnt think it was worth it, but I will add this in case someone tries to use it after my check-in

Reply:
fixed.